Mail/Views/Thread/ThreadView.swift

```diff
@@ -95,7 +95,7 @@ struct ThreadView: View {
                 .padding(.bottom, value: .regular)
                 .padding(.horizontal, value: .regular)
 
-                MessageListView(messages: thread.messages)
+                MessageListView(messages: thread.messages.freezeIfNeeded())
```

Comment:
We can pass `MessageListView(messages: messages.toArray()` directly so that we do it only once (and adapt MessageListView to accept an array instead of List)

Reply:
Good idea, it does clean the code quite nicely.